extensions/ai-gen/src/components/CreateImageForm.tsx

```diff
@@ -67,22 +153,48 @@ export function CreateImageForm(props: { draftValues?: CreateImageValues }) {
         </ActionPanel>
       }
     >
-      <Form.Description
-        title="OpenAI Image Generation"
-        text="Given a text prompt, generate a new image using the OpenAI DALL·E 2 image model"
-      />
-      <Form.TextArea
-        id="prompt"
-        title="Prompt"
-        placeholder="Describe the image you want to create"
-        info={`A text description of the desired image(s). The maximum length is ${MAX_CHARS} characters.`}
-        error={promptError}
-        onBlur={(event) => validatePrompt(event.target.value)}
-        onChange={(value) => validatePrompt(value)}
-        autoFocus={true}
-        storeValue={!draftValues?.prompt && storeValue}
-        defaultValue={draftValues?.prompt}
-      />
+      <Form.Description title="OpenAI Image Generation" text={descText} />
+      {!isVariation ? (
+        <Form.TextArea
+          id="prompt"
+          title="Prompt"
+          placeholder="Describe the image you want to create"
+          info={`A text description of the desired image(s). The maximum length is ${MAX_CHARS} characters.`}
+          error={promptError}
+          onBlur={(event) => validatePrompt(event.target.value)}
+          onChange={(value) => validatePrompt(value)}
+          autoFocus={!isVariation}
+          storeValue={!draftValues?.prompt && storeValue}
+          defaultValue={draftValues?.prompt}
+        />
+      ) : null}
+      {isVariation || isEdit ? (
+        <Form.FilePicker
+          id="images"
```

Comment:
Nit: Should this `id` be `image` since you have `allowMultipleSelection={false}`? Or are you using plural because the typing returns an array regardless?

Reply:
The latter. It's an array, so it should be plural

Reply:
Thought so 👍